1-exercises/E-for-loop/exercise2.js

```diff
@@ -28,6 +28,12 @@ const AGES = [
 
 // TODO - Write for loop code here
 
+for (let i = 0 ; i < WRITERS.length; i++){
+    console.log( WRITERS[i] + " is " + AGES[i] + " years old")
```

Comment:
```suggestion
    console.log(` ${WRITERS[i]} is ${AGES[i]} years old`);
```
Nothing wrong with the code but I would have used template literals